### DISCUSSION TITLE
Can't publish package: "Package must be in a Git repository before publishing", but it isn't true (I think)

### DISCUSSION THREAD

**Discussion by Luka108** [atom/atom#23083] [Q&A] [answered]
I have a simple problem: I created a simple package for Atom, but I can't publish it because when I launch `apm publish minor` I receive the error
`Package must be in a Git repository before publishing: https://help.github.com/articles/create-a-repo`
But in fact it is in a repo, and you can find it [here](https://github.com/Luka108/language-mindlang.git), where you can also check the `package.json`, to see if it's everything ok.
I'm struggling with this for two days, I hope I'll find a solution here.

Thank you in advance.

**Comment by UziTech** [comment]
Is it a private repo? I think it has to be a public repo for it to work.

**Comment by Luka108** [comment]
I don't know why the link in my question isn't working, but you can see here that the repo exists and it's public
![Screenshot_2021-10-08-08-16-34-974_com android chrome](https://user-images.githubusercontent.com/34659061/136507613-87e2ab66-ed9b-46a9-9ce0-5059e3a22e26.jpg)

I think my lack of knowledge of GitHub is the problem

EDIT: I solved the link not working (obviously not the publishing problem)

**Comment by Luka108** [comment]
I have an update

I somehow cloned the repo to my pc, and tried running the `apm publish minor` from the newly created folder.
It asked me the Atom token, so fantastic!
Then another error occurred, but I think I'm further ahead than before:
![Immagine 2021-10-12 185833](https://user-images.githubusercontent.com/34659061/136998671-3ef54770-801b-4e49-b7f5-ae8b4fe73dee.jpg)
As you can see, git asks me to tell him who I am, but I run the suggested instruction right before the last.
So, what to do?

**Comment by Luka108** [comment]
Now I discovered that my package has appeared [here](https://atom.io/packages/language-mindlang), and it's amazing!
But I can't find it by searching in Atom.
Anyway, I think that the content of the README file should be shown on Atom Package web page, so it's clear that there's something wrong.

**Comment by Luka108** [comment, marked as answer]
I solved the issue omitting the arg `--global` to say git who I am.

Thanks to all.
